src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/Event.java

```diff
@@ -26,6 +26,8 @@
     CREATE_CASE_SUMMARY("createCaseSummary"),
     REVERT_STATE_TO_AWAITING_RESPONDENT_EVIDENCE("revertStateToAwaitingRespondentEvidence"),
     GENERATE_HEARING_BUNDLE("generateHearingBundle"),
+    CUSTOMISE_HEARING_BUNDLE("customiseHearingBundle"),
+    ASYNC_STITCHING_COMPLETE("asyncStitchingComplete"),
```

Comment:
this event is not needed but it doesn't cause issues if we keep it here

Reply:
event removed